application/app/views/download_status/_files.html.erb

```diff
@@ -1,18 +1,37 @@
 <% files ||= [] %>
 <div id="schedule-downloads" class="card">
   <div class="card-header">
-    <h4><%= t('.scheduled_downloads') %></h4>
+    <h1><%= t('.scheduled_downloads') %></h1>
   </div>
   <div class="card-body bg-light px-0">
-    <div class="container files">
+    <div class="container files" role="table">
       <% files.each_with_index do |data, index| %>
-        <div class="row py-2 align-items-center <%= 'custom-stripe' if index.even? %>">
-          <div class="col-md-8">
+        <div class="row py-2 align-items-center <%= 'custom-stripe' if index.even? %>" role="row">
+          <!-- info for small screens -->
+          <div class="col-sm-12 d-md-none" role="cell">
             <div class="d-flex align-items-center">
-            <%= link_to(files_app_url(data.project.download_dir), class: 'btn btn-sm btn-outline-secondary me-2', title: t('.browse_file'), target: '_blank') do %>
-              <i class="bi bi-folder"></i>
+              <%= link_to(files_app_url(data.project.download_dir), class: 'btn btn-sm btn-outline-secondary me-2',
+                          title: t('.browse_file'), target: '_blank') do %>
+                <i class="bi bi-folder" aria-hidden="true"></i>
+                <span class="visually-hidden"><%= t('.browse_file_for', filename: data.file.filename) %></span>
+              <% end %>
+              <span class="me-2"><%= connector_icon(data.file.type) %></span>
+              <span class="me-2"><%= data.file.filename %></span>
+            </div>
+          </div>
+          <!-- end info for small screens -->
+
+          <!-- info for medium and large screens -->
+          <div class="col-md-8 d-none d-sm-none d-md-block" role="cell">
+            <div class="d-flex align-items-center">
+            <%= link_to(files_app_url(data.project.download_dir), class: 'btn btn-sm btn-outline-secondary me-2',
+                        title: t('.browse_file'), target: '_blank') do %>
+              <i class="bi bi-folder" aria-hidden="true"></i>
+              <span class="visually-hidden"><%= t('.browse_file_for', filename: data.file.filename) %></span>
             <% end %>
-              <div class="d-flex flex-column align-items-center fs-7 text-muted creation-date mx-2" title="<%= t('.creation_date') %>">
+              <div class="d-flex flex-column align-items-center fs-7 text-muted creation-date mx-2"
+                   aria-label="<%= t('.creation_date') %>"
+                   title="<%= t('.creation_date') %>">
                 <span><%= get_date(data.file.creation_date) %></span>
                 <span><%= get_time(data.file.creation_date) %></span>
               </div>
```

Comment:
I think that the inline creation data can be refactored and use:

`<%= render partial: '/shared/file_row_date', locals: { date: bundle.creation_date, title: t('.creation_date') } %>`

Take a look to see if it looks the same